include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -173,6 +173,13 @@ make_wrapped_policy(_Policy&& __policy)
         ::std::forward<_Policy>(__policy));
 }
 
+template <template <typename> class _NewKernelName, typename _Policy,
+          oneapi::dpl::__internal::__enable_if_host_execution_policy<_Policy, int> = 0>
+_Policy make_wrapped_policy(_Policy&& __policy)
+{
+    return __policy;
+}
+
```

Comment:
It seems like this was added to support some usages that have since been removed.  It seems like this is now dead code.

Is there any functionality for host policies that needs to be replaced in the `rebind_policy` calls?

Reply:
You are correct that this is just dead code now. I have removed it.